City-of-Helsinki/drupal-palvelukeskus
# UHF-9363: sentry

# [UHF-9363](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9363)


## What was done


* Enable sentry

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-9363-sentry`
  * `make fresh`
* Run `make drush-cr`

## How to test

* [ ] Verify that kibana is enabled
  * You can use this query to see palvelukeskus logs: `kubernetes.namespace_name:hki-kanslia-helfi-palvelukeskus-test AND  kubernetes.container_name:drupal`.

[UHF-9363]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9363?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
